server/server.go

```diff
@@ -66,7 +66,7 @@ func SetLogLevel(level log.LogLevel) {
 // that path params are testable.
 func NewRouter() *mux.Router {
```

Comment:
is server going to be exposed as lambda under a gateway?

Reply:
nope - it's a "real" server, running as an EKS container
(although, now you mention it, one could also conceive it the other way round, as a collection of Lambdas - a couple behind the G/W, and the other two connected to SQS, while Redis acts as the repository of state)

not how I've designed it so far, though (which is not to say it was The Right Way :laughing: )